src/components/basic/SelectList/index.tsx

```diff
@@ -28,20 +28,19 @@ import uniqueId from 'lodash/uniqueId'
 import isEqual from 'lodash/isEqual'
 import { useEffect, useState } from 'react'
 
+type Option = Record<string, string | number | null>
 interface SelectListProps extends Omit<TextFieldProps, 'variant'> {
-  // eslint-disable-next-line
-  items: any
+  items: Option[]
```

Comment:
same as above

Reply:
This change does not affect the portal app’s functionality